src/main/java/uk/gov/digital/ho/pttg/application/HmrcHateoasClient.java

```diff
@@ -229,6 +235,7 @@ private void checkForEmptyNormalizedName(HmrcIndividual normalizedIndividual) {
         return resource;
     }
 
+    @AbortIfBeyondMaxResponseDuration
```

Comment:
@Tim-Hammonds Please forgive me if I ask a stupid question, but I've never created custom annotations. Is the idea that every time we enter one of these annotated methods, we'll check the request time and throw an exception if it's too long?

Reply:
Exactly that.